src/java.base/share/classes/java/math/BigDecimal.java

```diff
@@ -3040,16 +3045,21 @@ public BigDecimal stripTrailingZeros() {
     // Comparison Operations
 
     /**
-     * Compares this {@code BigDecimal} with the specified
+     * Compares this {@code BigDecimal} numerically with the specified
      * {@code BigDecimal}.  Two {@code BigDecimal} objects that are
      * equal in value but have a different scale (like 2.0 and 2.00)
-     * are considered equal by this method.  This method is provided
+     * are considered equal by this method. Such values are in the same <i>cohort</i>.
+     *
+     * This method is provided
      * in preference to individual methods for each of the six boolean
      * comparison operators ({@literal <}, ==,
      * {@literal >}, {@literal >=}, !=, {@literal <=}).  The
      * suggested idiom for performing these comparisons is:
      * {@code (x.compareTo(y)} &lt;<i>op</i>&gt; {@code 0)}, where
      * &lt;<i>op</i>&gt; is one of the six comparison operators.
+
+     * @apiNote
+     * Note: this class has a natural ordering that is inconsistent with equals.
```

Comment:
This is the exact text recommended in java.lang.Comparable when a type's natural ordering is inconsistent with equals. The statement to that effect at the top of BigDecimal didn't use that exact wording

Reply:
Okay, I see.

Reply:
The note itself should be here, but it's demarcated with an `@apiNote` tag. This introduces a subhead "API Note:" in the rendered javadoc. Thus, it's not necessary to start the text with "Note:".

Reply:
My think here was to include the exact string "Note: this class has a natural ordering that is inconsistent with equals." in the text of BigDecimal since that is the phrase java.lang.Comparable recommends. This should improve grep-ability, etc. for such classes in the JDK. The class-level summary has a semantically equivalent statement using different wording, which left the compareTo method as a place to introduce the exact phrase. I use an @apiNote both because it is an informative statement and to prevent the text from necessarily being applied to BigDecimal subclasses. BigDecimal is not final and people can and sometimes do subclass it and those subclasses, in principle, could have a (different) natural order that was consistent with equals.